### ISSUE TITLE
2 tests in CI failing [2024-04-23]

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spreg#135]
Failures [here](https://github.com/pysal/spreg/actions/runs/8809267784/job/24179788657#step:5:1092):

```
FAILED spreg/tests/test_error_sp_sparse.py::TestGMCombo::test_model - AssertionError: 
Not equal to tolerance rtol=1e-05, atol=0

Mismatched elements: 1 / 8 (12.5%)
Max absolute difference: 3.80550383e-06
Max relative difference: 1.08244696e-05
 x: array([[ 2.520520e+00,  1.171817e-02],
       [ 1.505358e+00,  1.322319e-01],
       [-3.319092e+00,  9.031073e-04],
       [-4.685325e-01,  6.394038e-01]])
 y: array([[ 2.520516e+00,  1.171829e-02],
       [ 1.505360e+00,  1.322317e-01],
       [-3.319093e+00,  9.031031e-04],
       [-4.685305e-01,  6.394053e-01]])
FAILED spreg/tests/test_ml_error.py::TestMLError::test_ord - ValueError: zip() argument 2 is shorter than argument 1
```

**Comment by pedrovma** [comment]
How annoying can this be? 
Continuous Integration [#1165](https://github.com/pysal/spreg/actions/runs/9849201200)

**Comment by pedrovma** [comment]
Solved as of [CI #1203](https://github.com/pysal/spreg/actions/runs/10294114388).
